lib/detect-system.sh

```diff
@@ -31,6 +31,9 @@ parse_executable_arch()
 			_system_arch=aarch64;;
 		*ARM)
 			_system_arch=arm;;
+		 *RISC-V)
+                        _system_arch=riscv;;
+
```

Comment:
thanks, kindly use tab as indent

Reply:
> thanks, kindly use tab as indent

will do
Thanks

Reply:
you better rebase the code, so this commit can have the correct indent.